packages/alfa-dom/src/matches.ts

```diff
@@ -370,18 +420,29 @@ function matchesPseudoClass(
   element: Element,
   context: Node,
   selector: PseudoClassSelector,
-  options: MatchesOptions
+  options: MatchesOptions,
+  root: Selector
 ): boolean {
   switch (selector.name) {
     // https://www.w3.org/TR/selectors/#scope-pseudo
     case "scope":
       return options.scope === element;
 
+    // https://www.w3.org/TR/css-scoping/#host-selector
```

Comment:
Upon closer inspection the current TR for CSS Scoping is horribly out of date so let's link to: https://drafts.csswg.org/css-scoping/#host-selector. I think there's a couple more references in other files as well.